test/Python/src/docker/docker_nonfederated/PingPongContainerized.lf

```diff
@@ -0,0 +1,38 @@
+/**
+ * Basic benchmark from the Savina benchmark suite that is
+ * intended to measure message-passing overhead.
+ * This is based on https://www.scala-lang.org/old/node/54
+ * See https://shamsimam.github.io/papers/2014-agere-savina.pdf.
+ * 
+ * Ping introduces a microstep delay using a logical action
+ * to break the causality loop.
+ * 
+ * To get a sense, some (informal) results for 1,000,000 ping-pongs
+ * on my Mac:
+ * 
+ * Unthreaded: 97 msec
+ * Threaded: 265 msec
+ * 
+ * There is no parallelism in this application, so it does not benefit from being
+ * being threaded, just some additional overhead.
+ * 
+ * These measurements are total execution time, including startup and shutdown.
+ * These are about an order of magnitude faster than anything reported in the paper.
+ * 
+ * @author Edward A. Lee
+ */
+target Python {
+    fast: true,
+    docker: true
+};
+
+import Ping from "../../PingPong.lf"
```

Comment:
This file should also be in `docker`, not `docker/docker_nonfederated`, I think.